teams/edition.toml

```
@@ -0,0 +1,39 @@
+name = "edition"
+subteam-of = "launching-pad"
+
+[people]
+leads = [
+    "traviscross",
+    "ehuss",
+]
+members = [
+    "traviscross",
+    "ehuss",
+]
+alumni = [
+    "bstrie",
+    "m-ou-se",
+]
+
+[permissions]
+perf = true
+crater = true
+dev-desktop = true
+bors.rust.review = true
+
+[[github]]
+orgs = ["rust-lang", "rust-lang-nursery"]
```

I think we're moving towards closing this org down, and I don't think there is anything there we need.

```suggestion
orgs = ["rust-lang"]
```

---

OK, updated.